tests/build_test.sh

```diff
@@ -238,6 +238,18 @@ function test_kernel_build_menu_cross_compilation_flags()
   assertEquals "($LINENO)" "$expected_result" "$output"
 }
 
+function test_kernel_build_clean() {
+  local expected_result
+  local output
+
+  build_config=()
+  parse_configuration "$SAMPLES_DIR/build_no_log.config" build_config
+
+  output=$(kernel_build 'TEST_MODE' --clean 2> /dev/null)
+  expected_result="make clean"
```

Comment:
Use single quotes.

Reply:
in the `expected_result` variable or `parse_configuration`?